### ISSUE TITLE
False positive for is_a()

### ISSUE THREAD

**Issue by FlyingDR** [kalessil/phpinspectionsea#850]
PHPStorm 2017.3.5, PHP Inspections 2.3.18

`Callable parameter usage violates definition` inspection have false positive on passing `string` argument for `is_a()` function. Passing string argument is [allowed](https://secure.php.net/manual/en/function.is-a.php#refsect1-function.is-a-changelog) since PHP 5.3.9 and is controlled by `$allow_string` argument. However this code reports definition violation:
```php
/**
 * @param string $class
 * @return bool
 */
function test($class)
{
    return is_a($class, \stdClass::class, true);
}
```

**Comment by kalessil** [comment]
Fixed
